.github/workflows/run_admission_test.yml

```diff
@@ -9,15 +9,6 @@ jobs:
       group: ${{ github.workflow }}-${{ github.ref }}
       cancel-in-progress: true
     steps:
-      - name: Checkout noobaa-core
-        uses: actions/checkout@v3
-        with:
-          repository: 'noobaa/noobaa-core'
-          path: "noobaa-core"
-          # Freeze the version of core
-          # to avoid a failed run due to code changes in the core repo.
-          # Need to update the commit once in a while
-          ref: c97e4ddd2e5fcc110a3242c224eeea97d228c6eb
```

Comment:
@baum, Is it a temporary change?
We freeze the commit of noobaa-core on purpose because we want the CI to check only the Operator changes.

Reply:
Master Root Key changes interface with the core in regards with sharing the key, that SHA/ref would of core would not able to run. 

Generally, the admission test froze the version of the core in the GitHub action, possibly to ensure stability during testing. However, the comment suggests that it is necessary to update the commit periodically to keep up with changes or fixes in the codebase.

```shell
          # Need to update the commit once in a while
```

Use the operator's default NooBaa core image, which may not be the most recent version of the NooBaa core but is presumably stable enough to use.

Reply:
Extracted admission test change into a separate [PR](https://github.com/noobaa/noobaa-operator/pull/1076).